### ISSUE TITLE
🔦 Feature: Display M2M Tables name near every m2m relationship

### ISSUE THREAD

**Issue by MichaelCereda** [nocodb/nocodb#8553]
### Please confirm if feature request does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

It's often very useful to create views or routines on other schemas that link directly to tables in Nocodb.
It's getting progressively harder to see what is the name of the m2m table that links two other tables.

### Suggested Solution

make the m2m table related to a column visible in the dropdown menu of the said column.
<img width="212" alt="image" src="https://github.com/nocodb/nocodb/assets/107390/cd25c844-c92e-4ce0-a4a4-3144fab7f968">
or as part of the "Details > Relations" section.
<img width="525" alt="image" src="https://github.com/nocodb/nocodb/assets/107390/06e6f51c-b372-4751-8338-f32a2e039a33">

Another alternative would be to allow another checkbox in the relations graph `[] Show M2M tables.`
<img width="295" alt="image" src="https://github.com/nocodb/nocodb/assets/107390/4d222c8a-9a49-46d1-b00c-2ff5adafa097">
 
I think the cleanest solution would be the latter, but it might be more involved than the other two.

**Comment by dstala** [comment]
Base settings has an option `Show M2M tables`
![Screenshot 2024-05-24 at 7 05 04 PM](https://github.com/nocodb/nocodb/assets/86527202/9f234582-af6f-406c-9bf0-1b39e59a101c)

Is this what you are looking for? Why do you wish to access m2m relationship table?

**Comment by MichaelCereda** [comment]
Yes... that would show the m2m in the list of the tables (Sidebar).
But I cnnot see the m2m associated to a specific relationship like I did before....
![image](https://github.com/nocodb/nocodb/assets/107390/76b43207-7634-4654-937c-ca9456138cae)
the many to many above was also showing a m2m prefixed table in the "system fields" before the recent updates.
now it's very hard to find which one of the m2m tables is joining Table1 with Table2.

Is there a way to solve that? maybe I can be pointed to an internal table that does the mapping.
The current m2m tables also don't use foreign keys so I cannot check the definition to understand what they are pointing to.
Thank you

**Comment by dstala** [comment]
is now supported in latest release
